include/CppUTest/CppUTestConfig.h

```diff
@@ -199,6 +199,27 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Support for "long long" type */
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
I haven't had a chance to try it yet (soon!), but I think I can make it so that you can centrally install one version with `USE_LONG_LONG` enabled, and then link against it both with or without `USE_LONG_LONG`.  Though it requires a couple of minor shenanigans.

If you centrally install a version without longlongs then you'll still get link errors if you try to `USE_LONG_LONG` in the app, but that seems reasonable too.